Yoast/wordpress-seo
# P3-77 Read option for SEMrush integration to show or hide the "Get related keyphrases" button

## Context


* P3-73 added an "Integrations" tab with a toggle to enable/disable the SEMrush feature. We need to read its value to show/hide the button that opens the SEMrush modal.

## Summary


This PR can be summarized in the following changelog entry:

* Show/hide the "Get related keyphrases" button in the metabox and in the sidebar, based on the SEMrush integration option.

## Relevant technical choices:

* Disabling integration just hides the button (= it's not rendered). No other action (e.g. removing auth tokens etc.) is performed.

## Test instructions

This PR can be tested by following these steps:

* Pull the branch and build
* Visit SEO Settings, "Integrations" tab, to check that the SEMrush integration toggle is set to "On"
* Edit a new post or an existing one
* Observe that in the metabox or in the sidebar the "Get related keyphrases" button is present and working (the modal opens only if you have inserted a keyphrase)
* Visit SEO Settings, "Integrations" tab, set the SEMrush integration toggle to "Off" and save
* Edit a new post or an existing one
* Observe that in the metabox or in the sidebar the "Get related keyphrases" button is now absent.

## UI changes

## Documentation

## Quality assurance


Fixes [P3-77]
